src/protocol/cdp.ts

```diff
@@ -70,18 +72,14 @@ export type GetSessionResult = {
   session?: Protocol.Target.SessionID;
 };
 
-export type EventReceivedEvent = {
-  method: EventNames;
-  params: EventParameters;
-};
-
-export type EventParameters<
-  EventName extends keyof ProtocolMapping.Events = keyof ProtocolMapping.Events,
-> = {
-  event: EventName;
-  params: ProtocolMapping.Events[EventName][0];
-  session: Protocol.Target.SessionID;
+export type EventDataFor<EventName extends keyof ProtocolMapping.Events> = {
+  method: `cdp.${EventName}`;
+  params: EventParametersFor<EventName>;
 };
 
-// XXX: Support `cdp` (all events)
```

Comment:
To be clear: Is `cdp` (=all events) supported with this PR?

Reply:
It seems the answer is no, and that you just moved the TODO to the other file. Just checking.